iterations_runners/iterations_runner.java

```diff
@@ -223,14 +223,15 @@ public static void main(String args[]) throws
             Arrays.fill(mperfCounts[core], 0);
         }
 
+        long checksum = 0;
         for (int i = 0; i < iterations; i++) {
             if (debug) {
                 System.err.println("[iterations_runner.java] iteration: " + (i + 1) + "/" + iterations);
             }
 
             // Start timed section
             IterationsRunner.JNI_krun_measure(0);
-            ke.run_iter(param);
+            checksum += ke.run_iter(param, checksum);
```

Comment:
For this one, we can use `checksum =` but the checksum will then grow linearly, instead of exponentially, thoughts?